### PR TITLE
[Docs] Adding section to docs about SASS and LESS

### LINKED ISSUE #1612: [doc]: Webpack less configuration
Please provide web-pack configuration file for less and css loader which should have scope of venia-ui repository

### LINKED ISSUE #1759: [doc]: Webpack less configuration
Currently pwa repository provide with cssModule integration. If any user want to add a less file in their project he/she will be get better idea  by reading this document.

### PR DESCRIPTION
**Pull request by LucasCalazans**
## Description

New devdocs section on the tutorials about how to configure SASS and LESS.

## Related Issue
Closes #1759
Closes #1612 

## Checklist

* I have updated the documentation accordingly
* I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by devops-pwa-codebuild** [comment]
**Performance Test Results**

The following fails have been reported by WebpageTest. These numbers indicates a possible performance issue with the PR which requires further manual testing to validate.

https://pr-2316.pwa-venia.com/venia-tops.html : LH Performance Expected 0.75 Actual 0.66

File: pwa-devdocs/src/_data/tutorials.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ entries:
     - label: Create a TagList component
       url: /tutorials/create-taglist-component/
 
+    - label: Configure SASS or LESS
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/_data/tutorials.yml]: Changed in 13a718d.

File: pwa-devdocs/src/_data/tutorials.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ entries:
...
+      url: /tutorials/configure-sass-less/
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/_data/tutorials.yml]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+---
+title: Configure SASS or LESS
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+---
+
+This tutorial provides the steps to configure SASS or LESS in your project.
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+With this configuration, you will be able to use these preprocessors with [CSS Modules][].
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+const MyComponent = () => (
+    <div className={defaultClasses.root}>
+        <button className={defaultClasses.button}>My Component</button>
+    </div>
+);
+
+export default MyComponent;
+```
+
+And create the `myComponent.scss` file:
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Moved in 13a718d, but double check the wording. Looked like you suggested removing adverbs, so tried to stick with that tone here.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+.button {
+    color: $button-color;
+
+    &:hover {
+        color: $button-color-hover;
+    }
+}
+```
+
+## Configuring LESS
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+### Step 1. Install dependencies
+
+To configure LESS, you need to install these dev dependencies:
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+```sh
+yarn add --dev less-loader less
+```
+
+### Step 2. Modify the Webpack configuration
+
+Then edit the `webpack.config.js` and add a new rule to clientConfig
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Changed in 13a718d.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+const MyComponent = () => (
+    <div className={defaultClasses.root}>
+        <button className={defaultClasses.button}>My Component</button>
+    </div>
+);
+
+export default MyComponent;
+```
+
+And create the `myComponent.less` file:
```
[the remark this answers is not in the document]
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: Movied in 13a718d, double check wording 🙂

**Comment by tjwiebell** [comment]
@jcalcaben - Addressed your feedback, and updated code samples to some recent changes to webpack.config. LMK if there's anything else to get this merged.

File: pwa-devdocs/src/_data/tutorials.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcalcaben) [reply pwa-devdocs/src/_data/tutorials.yml]: The `configure-sass-less` directory needs to be changed to `enable-sass-less-support` so the navigation link doesn't break.

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
++           'less-loader'
++       ]
++   });
+```
+
+### Step 3. Create a LESS file and import it on your component
+
+Create the `myComponent.less` file:
+
+```
```
Comment (jcalcaben) [inline pwa-devdocs/src/tutorials/configure-sass-less/index.md:155]:
```suggestion
```less
```

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+const MyComponent = () => (
+    <div className={defaultClasses.root}>
+        <button className={defaultClasses.button}>My Component</button>
+    </div>
+);
+
+export default MyComponent;
+```
+
+[CSS modules]: <{%link technologies/basic-concepts/css-modules/index.md %}>
```
Comment (jcalcaben) [inline pwa-devdocs/src/tutorials/configure-sass-less/index.md:187]:
```suggestion
[css modules]: <{%link technologies/basic-concepts/css-modules/index.md %}>
```

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md (hunk 11 above)
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: 3ecc22b

File: pwa-devdocs/src/tutorials/configure-sass-less/index.md (hunk 12 above)
Comment (tjwiebell) [reply pwa-devdocs/src/tutorials/configure-sass-less/index.md]: 3ecc22b

**Comment by dpatil-magento** [comment]
Ran `yarn develop` and verified new page exists.

**Pull request merged**